.circleci/config.yml

```diff
@@ -118,10 +118,16 @@ jobs:
           name: Run tests
           command: |
             source /home/firedrake/firedrake/bin/activate
-            coverage run --source=firedrake_adjoint,pyadjoint -m pytest tests/firedrake_adjoint/ tests/pyadjoint/
+            coverage run --source=pyadjoint -m pytest tests/pyadjoint/
             coverage report
             coverage html
 
+      - run:
+          name: Run firedrake adjoint tests
+          command: |
+            source /home/firedrake/firedrake/bin/activate
+            coverage run python3 -m pytest /home/firedrake/firedrake/src/firedrake/tests/firedrake/adjoint
```

Comment:
@connorjward, do you think this is the right script to run firedrake adjoint tests in pyadjoint CI?

Reply:
I think `coverage run` won't be necessary here because `coverage report` and `coverage html` are run in the step above. Also it doesn't make sense to measure test coverage for a suite of external tests. I think you just want
```
python3 -m pytest /home/firedrake/firedrake/src/firedrake/tests/firedrake/adjoint
```